invisible_cities/core/core_functions.py

```diff
@@ -77,6 +82,53 @@ def in_range(data, minval=-np.inf, maxval=np.inf, left_closed=True, right_closed
     return lower_bound & upper_bound
 
 
+def all_in_range(data         : np.ndarray,
+                 minval       : float   ,
+                 maxval       : float   ,
+                 display_name : str = '',
+                 strictness   : Strictness = Strictness.stop_proccess,
+                **kwargs)->bool:
+    """
+    Checks whether input values are all inside the interval (minval, maxval).
+
+    Parameters
+    ----------
+    data : np.array
+        Input array to check.
+    minval: float
+        Lower limit of the interval.
+    maxval: float
+        Upper limit of the interval.
+    display_name: string
+        Name for the message to print if exception raises.
+    strictness: Strictness
+        If 'warning', function returns a False if the criteria
+        is not match. If 'stop_proccess' it raises an exception.
+    Returns
+    ----------
```

Comment:
```suggestion
    Returns
    -------
```

(matching in length)